service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -231,14 +232,22 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
   @Override
   public ResponseEntity<AbortRunSetResponse> abortRunSet(UUID runSetId) {
     AbortRunSetResponse aborted = new AbortRunSetResponse();
-    List<UUID> dummyRunsList = new ArrayList<>();
+    aborted.runSetId(runSetId);
 
-    dummyRunsList.add(UUID.randomUUID());
-    dummyRunsList.add(UUID.randomUUID());
+    List<Run> runningWorkflows =
+        runDao.getRuns(new RunDao.RunsFilters(runSetId, NON_TERMINAL_STATES));
+    List<String> submittedAbortWorkflows = new ArrayList<>();
 
-    aborted.runSetId(runSetId);
+    for (Run run : runningWorkflows) {
```

Comment:
To think about in the future:
- We will probably need to consider what happens when we overflow the number of workflows we can abort all at the same time